### PR TITLE
Additional config properties for org auto-config

### PR DESCRIPTION
**Pull request by austinkelleher**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -71,3 +73,9 @@ export const invocationConfig: IntegrationInvocationConfig<IntegrationConfig> =
       };
     },
   };
+
+export {
```
Comment (austinkelleher) [inline src/index.ts:77]: Using these in managed GCP integration

File: src/types.ts [hunk 2]
```diff
@@ -9,6 +9,19 @@ export type IntegrationStepContext =
 
 export interface SerializedIntegrationConfig extends IntegrationInstanceConfig {
```
Comment (austinkelleher) [inline src/types.ts:10]: Added these properties for org auto-config

File: package.json [hunk 3]
```diff
@@ -48,6 +48,7 @@
   "dependencies": {
     "@lifeomic/attempt": "^3.0.0",
     "gaxios": "^4.2.1",
-    "googleapis": "^73.0.0"
+    "google-auth-library": "^7.1.0",
```
Comment (austinkelleher) [inline package.json:51]: This package was being used and should have been a direct dep. It was technically being installed as a transitive dep

File: src/google-cloud/client.ts [hunk 4]
```diff
@@ -29,12 +23,6 @@ export type PageableGaxiosResponse<T> = GaxiosResponse<
   }
 >;
 
-export type GoogleClientAuth =
```
Comment (austinkelleher) [inline src/google-cloud/client.ts:32]: The Google auth client doesn't export the TS type for this. I did some casting below to simplify things a bit even though it's ugly and sucks.

**Review by aiwilliams — APPROVED** [review]

**Pull request merged**